pg-core/src/java/org/pg/util/SocketTool.java

```diff
@@ -11,11 +12,16 @@
 
 public class SocketTool {
 
```

Comment:
Here, there is a thing that looks so-so: coupling SocketTool and Config classes. Is that possible to fix it such that SocketTool accepts options as primitives?

Reply:
Okay! I've moved it into the class itself